packages/rum-core/src/domain/view/viewCollection.ts

```diff
@@ -18,6 +18,7 @@ export function startViewCollection(
   configuration: RumConfiguration,
   location: Location,
   domMutationObservable: Observable<void>,
+  pageOpenObserable: Observable<void>,
```

Comment:
Nit: "Obserable"

Reply:
To be consistent with other places in the code, it might be better to name this `windowOpenObservable`.

Reply:
Done using @thomas-lebeau update